build/CommonBuild.Private.props

```diff
@@ -58,4 +58,5 @@
     <PropertyGroup Label="Configuration">
         <SpectreMitigation>Spectre</SpectreMitigation>
     </PropertyGroup>
+    <Import Project =".\CommonBuild.Cpp.props" Condition="'$(MSBuildProjectExtension)'=='.vcxproj'"/>
```

Comment:
><Import Project =".\CommonBuild.Cpp.props" Condition="'$(MSBuildProjectExtension)'=='.vcxproj'"/> [](start = 4, length = 97)

There is already a ClrInstrumentationEngine.Cpp.props that is meant to be included by vcxproj files.